diff -up multitail-6.4.2/Makefile.orig multitail-6.4.2/Makefile
--- multitail-6.4.2/Makefile.orig	2015-12-23 22:58:28.000000000 -0500
+++ multitail-6.4.2/Makefile	2015-12-23 22:58:41.000000000 -0500
@@ -30,8 +30,6 @@ install: multitail
 	cp multitail $(DESTDIR)$(PREFIX)/bin
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
 	cp multitail.1 $(DESTDIR)$(PREFIX)/share/man/man1/multitail.1
-	mkdir -p $(DESTDIR)$(PREFIX)/share/doc/multitail-$(VERSION)
-	cp *.txt INSTALL manual*.html $(DESTDIR)$(PREFIX)/share/doc/multitail-$(VERSION)
 	#
 	### COPIED multitail.conf.new, YOU NEED TO REPLACE THE multitail.conf
 	### YOURSELF WITH THE NEW FILE
@@ -56,7 +54,6 @@ uninstall: clean
 	rm -f $(DESTDIR)$(PREFIX)/bin/multitail
 	rm -f $(DESTDIR)$(PREFIX)/share/man/man1/multitail.1.gz
 #	rm -f $(CONFIG_FILE)
-	rm -rf $(DESTDIR)$(PREFIX)/share/doc/multitail-$(VERSION)
 
 clean:
 	rm -f $(OBJS) multitail core gmon.out *.da ccmultitail
